config.go

```diff
@@ -14,7 +14,9 @@ type Config struct {
 	username     string
 	password     string
 	runtimePath  string
+	dataPath     string
 	locale       string
+	reuseData    bool
```

Comment:
May be enough to just use dataPath to infer the reuse of data? That way we can try and keep configuration to a minimum?

Reply:
good idea